_includes/markdown/Web-Vitals.md

```diff
@@ -0,0 +1,293 @@
+9. Hover over each "Location" / "Size" label and Chrome will highlight the offending element on the page.
+
+As an alternative, you can also diagnose Layout Shifts on the page by:
+
+1. Open _Google Chrome_
+2. Open _Chrome DevTools_
+3. Hit `⌘ ⇧ P` to open the actions console.
+4. Start typing "Rendering" until the prompt suggests: "Show Rendering", hit Enter.
+5. A dialog will appear at the bottom of the DevTools window.
+6. Check "Layout Shift Regions" and refresh the page.
+7. All elements that have been identified as triggering a layout shift will be highlighted.
+
+### How to fix Cumulative Layout Shift
+
+Elements that cause CLS can be easily fixed in some instances. As a general rule of thumb ensure that:
+
+* All images loaded on the site have a `width` and a `height` attribute. This is because HTML gets parsed before CSS and the browser will reserve space if it knows the dimensions and aspect-ratio of the image.
+* Ensure that ads, iframes and other embeds have a `width` and `height` attribute.
+* As a best practice, do not insert dynamic content into the site without the user performing an action to recieve it, ie "load more" or "click".
+* Ensure that you have a Web Font Loading strategy in place.
+* When animating CSS properties, ensure that you animate `transform` properties rather than `box-model` properties to prevent reflow and layout changes in the browsers [Critical Rendering Path](https://developers.google.com/web/fundamentals/performance/critical-rendering-path)
+
+#### _Handling Ad Sizes_
+When it comes to ads, its important that slot sizes are consistent in order to prevent CLS. Ads are generally difficult to predict considering that Ad Servers can serve ads at different heights and widths depending on impression data. There are a number of ways to mitigate this:
+
+1. Speak to Audience and Revenue or your Ad Ops team and see if its possible to ensure that ad units are served at more consistent sizes based on impression analytics. 
+2. If you can achieve more consistent ad sizes, you can reserve space for ad slots by using the `min-height` CSS property: `<div id="ad-slot" style="min-height: 250px;"></div>`
+3. If you want to dynamically set the height and width of ads once the `GPT.js` script has ad data you can use the following function:
+
+```
+googletag.pubads().addEventListener('slotRenderEnded', function(event) {
+	var size = event.size;
+	if(size === null) return;
+	var slot = event.slot;
+	var slotDiv = document.getElementById(slot.getSlotElementId());
+
+	if (size[0] > slotDiv.clientWidth) {
+		slotDiv.style.width = size[0] + 'px';
+	}
+
+	if (size[1] > slotDiv.clientHeight) {
+		slotDiv.style.height = size[1] + 'px';
+	}
+});
+```
+
+4. A combination of both 2 and 3 has yielded great improvement to CLS scores.
+5. Consider using `googletag.pubads().collapseEmptyDivs();` to ensure that ad slots that probably wont fill take up no height and width on the page.
+
+#### _Handling Web Fonts_
```

Comment:
@dainelmawer 
Before you publish this section - I recommend connecting with Ian Del Guidice as he has been doing some font loading experiments relating to performance.

Reply:
I'll reach out to Ian!

Reply:
@joesnellpdx I spoke to Ian about this - we were pretty much aligned! No need to add anything more here!